README.md

```diff
@@ -32,6 +32,20 @@ $ jshint app/assets/javascripts
 ## Deployment
 
 ### Development Mode
+
+You need multiple tools for the full development cycle. Here is the rundown for Ubuntu 14.04 LTS as a reference. If you run into errors with `bundle`, the most likely reason is that the database servers are not properly installed with dev headers/libs.
+
+* nodejs
```

Comment:
I'm fairly sure i needed to install it on the fresh VM. I think thats besides the point as this should be a reference in general for linux distros, even if happen to be using Ubuntu and `apt`. I think it should be explicitly stated that this tool is needed.